### PR TITLE
[Method References 2] LPS-171231 Convert method reference to ServiceTrackerMap in class MultiLanguageKaleoTaskAssignmentSelector

### PR DESCRIPTION
**Pull request by W2jx**
resend

previous: https://github.com/liferay-core-infra/liferay-portal/pull/4688

### CODE REVIEW

**Comment by W2jx** [comment]
ci:test:workflow

**Comment by catarineaguiar** [comment]
no regressions found.

**Comment by W2jx** [comment]
no conflicts(6/6 8:46)

**Comment by W2jx** [comment]
no conflicts(6/7 8:27)

File: modules/apps/portal-workflow/portal-workflow-kaleo-runtime-impl/src/main/java/com/liferay/portal/workflow/kaleo/runtime/internal/assignment/MultiLanguageKaleoTaskAssignmentSelector.java [hunk 1]
```diff
@@ -81,43 +83,39 @@ public Collection<KaleoTaskAssignment> getKaleoTaskAssignments(
...
+		_serviceTrackerMap = ServiceTrackerMapFactory.openSingleValueMap(
+			bundleContext, ScriptingAssigneeSelector.class,
+			"(scripting.language=*)",
+			(serviceReference, emitter) -> {
+				String className = ClassUtil.getClassName(
+					bundleContext.getService(serviceReference));
+
+				try {
+					for (String scriptingLanguage :
+							_getScriptingLanguages(serviceReference)) {
```
[the remark this answers is not in the document]
Comment (W2jx) [reply modules/apps/portal-workflow/portal-workflow-kaleo-runtime-impl/src/main/java/com/liferay/portal/workflow/kaleo/runtime/internal/assignment/MultiLanguageKaleoTaskAssignmentSelector.java:97]: Sorry, It's my bad, I will modify it.

**Pull request closed without merging**